distribution/sdk/artemis-sandbox/src/main/java/sdk.example.route/CoreProducerConsumerExample.java

```diff
@@ -59,6 +63,11 @@ private void createCamelContext() throws Exception {
         ActiveMQJMSClient.createConnectionFactory(
             "(tcp://0.0.0.0:5672,tcp://0.0.0.0:61617)?ha=true;sslEnabled=true;enabledCipherSuites=TLS_DHE_RSA_WITH_AES_128_GCM_SHA256,TLS_DHE_RSA_WITH_AES_128_CBC_SHA256,TLS_DHE_RSA_WITH_AES_128_CBC_SHA,TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256;enabledProtocols=TLSv1.1,TLSv1.2",
             "name");
+    if (factory == null) {
```

Comment:
Does not seem to be recognized by my SonarLint scan, and concerns of a null factory object was recognized in the Coverity scan. This can probably be removed in the future if other scans verify that it's not needed.